reindex.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
```

recommend adding `set -e` to the top of any shell script used as part of ci, so that silent failures don't end up with false positive builds.

---

This is for local dev, I'll delete it when the PR is ready. Which makes me remember that I need to document how to setup in local. I'll add few extra tasks in the PR description.